src/test/resources/manual-test/readme.md

```diff
@@ -0,0 +1,14 @@
+JSON files for manual tests
+---------------------------
+
+* Launch a virtual machine with `vagrant up` in the root of the project
+* Create a deposit, for details see `docs/api.html`
+* Use the returned UUID to replace XXX in the following command.
+
+    curl -i  -H 'Content-Type: text/plain'  --data-binary "@XXX.json" -X POST -u user001:user001 'http://test.dans.knaw.nl:20190/deposit/XXX/datasetmetadata'
```

Comment:
unrelated to previous comment: what does this `@` do in `"@XXX.json"`

Reply:
see man pages of curl

Reply:
👍
> If you start the data with the letter @, the rest should be a filename.